### ISSUE TITLE
Using Dialup with Windows - untrusted X11 forwarding

### ISSUE THREAD

**Issue by webhead404** [preludeorg/operator-support#138]
After following the steps to get dialup installed when I try to start a session I am presented with this error:

```untrusted x11 forwarding setup failed xauth key data not generated```

![xauth_issue](https://user-images.githubusercontent.com/8658818/112512029-fadb8780-8d60-11eb-844f-3bde6562c1f1.jpg)

**Comment by khyberspache** [comment]
Can you try removing the extra `2` from the `22222`:

`ssh -XC -p 2222 vagrant@localhost operator`

**Comment by khyberspache** [comment]
Per offline convo, probably due to the lack of X11 server on the windows host.

**Comment by webhead404** [comment]
I installed `XMing` and had to utilize Putty to get X11 forwarding working. I found a few issues with configuration

1. Vagrant didn't create a `.Xauthority` file in the vagrant home directory so the first X11 test failed.

2. When running `operator` from the PuTTy window there is an access denied message that the vagrant user doesn't have access to `./config/Operator/staging` (that directory didn't exist either upon first execution, nor were there any files in it upon execution.) I am assuming this directory is created for the sign in process in Operator.

3. I was able to get X11 forwarding working with `xclock` but not Operator. I couldn't find any additional logging methods for reference to see if there was something else I can hand off to troubleshoot this issue on Windows. 

Not sure how much else can be done or where the choke point is in relation to getting this working on Windows but eager to help figure it out for folks!

**Comment by khyberspache** [comment]
Thank you for the extremely detailed technical approach. I'm going to poke at this over the weekend then probably log it as a feature request pending additional user requests. 

I'm thinking it might be an X11 server feature thing on windows and it might boil down to X11 implementations.

**Comment by webhead404** [comment]
After much testing with various SSH clients I found another XServer under Cgywin. This has worked flawlessly! 

I will try to document the steps later to get it installed. I was able to make it work with the various SSH clients by tweaking settings and variables which I think is going to be a trial and error for most. 🤷

Operator still has errors related to the vagrant user. It isn't able to create directories under .config/Operator for some strange reason.
